### PR TITLE
enhance: support upsert autoid==true

### PR DESCRIPTION
**Pull request by smellthemoon**
related with: #29258

### CODE REVIEW

**Comment by yiwen92** [comment]
@czs007 could you pls accelerate the review? thx

**Comment by Seaiii** [comment]
@czs007  could you pls accelerate the review? thx

File: internal/proxy/proxy_test.go [hunk 1]
```diff
@@ -642,6 +642,22 @@ func TestProxy(t *testing.T) {
 		}
 	}
 
+	constructCollectionUpsertRequestWithPK := func() *milvuspb.UpsertRequest {
```
Comment (xiaofan-luan) [inline internal/proxy/proxy_test.go:645]: why not simply modify constructCollectionUpsertRequest by adding a input

**Pull request merged**